servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultHttpServerBuilder.java

```diff
@@ -261,7 +253,8 @@ public HttpServerBuilder bufferAllocator(final BufferAllocator allocator) {
 
     @Override
     public Single<HttpServerContext> listenStreaming(final StreamingHttpService service) {
```

Comment:
Consider adding a test when users specify a custom strategy with one flag on the builder and another flag on the filter. Assert that the resulting strategy has both offloading flags set for any `listen*` method.